venv/duplicates_arr.py

```diff
@@ -0,0 +1,9 @@
+my_list = [2, 4, 6, 8, 13, 16, 2, 6, 2, 'w', 's', 'w', 16]
+res_list = []
+for item in my_list:
+    res = my_list.count(item)
+    if res == 2:
+        res_list.append(item)
+res_list = set(res_list)  # .list(res_list) так почему нельзя?
+res_list = list(res_list)
```

Comment:
Это все можно значительно упростить
```suggestion
for item in set(my_list):
    if my_list.count(item) > 1:
        res_list.append(item)
```